.travis.yml

```diff
@@ -3,29 +3,35 @@ language: go
 
 go:
   - "stable"
-  - "1.10.4"
+  - "1.11.x"
+  - "1.10.x"
   - "1.9.x"
   - "1.8.x"
 
 matrix:
+  include:
+    - go: "stable"
+      env: GOLINT=true
   allow_failures:
     - go: tip
   fast_finish: true
 
-before_script:
-  - go get -u golang.org/x/lint/golint
+
+before_install:
+  - if [ ! -z "${GOLINT}" ]; then go get -u golang.org/x/lint/golint; fi
 
 script:
   - go test --race ./...
 
 after_script:
   - test -z "$(gofmt -s -l -w . | tee /dev/stderr)"
```

Comment:
This looks good. Thanks Brian!

It does make me wonder if we need to run `gofmt` or `go vet` on anything but the stable release too, but for now it doesn't hurt.